docs/ops.html

```diff
@@ -866,7 +866,7 @@ <h4><a id="remote_jmx" href="#remote_jmx">Security Considerations for Remote Mon
         <td></td>
       </tr>
       <tr>
-        <td># of under replicated partitions (|ISR| &lt |all replicas|)</td>
+        <td># of under replicated partitions (|original replicas| - |ISR| &gt 0)</td>
```

Comment:
What are people's thoughts if we say `non-reassigning replicas`?

From a first read of the docs, I don't believe people will understand what "original replicas" means

Reply:
Thanks, I was trying to find a better term, I think this is it.